packages/devextreme/js/__internal/ui/m_rating.ts

```diff
@@ -0,0 +1,12 @@
+import registerComponent from '@js/core/component_registrator';
```

Comment:
Kindly remove prefix m_ from this file. We shouldn't ignore eslint errors for new components.